rest-server/auth-server/src/main/resources/application.conf

```diff
@@ -9,10 +9,39 @@ auth-server {
 
   mongo-db {
     uri = "mongodb://localhost:27017"
-    db-name = "harness_auth"
+    db-name = "harness_auth_test"
   }
 
+  clients = [{
+    id = "harness"
+    password = "mwjmo4fwC9qFW081gm5vwsmAC4Z2747XEU8eeV6dzUzjW"
+  }]
+
+  role-sets = [
+    {
+      id = "admin"
+      roles = [
+        "user_create",
+        "user_permissions",
+        "engine_create"
+      ]
```

Comment:
and admin can do anything to any resource, this doesn't seem to capture that. For instance the admin cannot do what a client can do.

Reply:
So, engine_create shouldn't be at any roleset? Should this role be granted manually later?

Reply:
@pferrel now we managing roles with roleSets. What roleSet should contain engine_create role?
Looks like it's neither admin nor client, so we need another roleSet?

Reply:
@pferrel changed rolesets, admin now contains all the roles